src/main/kotlin/com/nikitahohulia/listeningplatform/controller/UserController.kt

```diff
@@ -0,0 +1,41 @@
+package com.nikitahohulia.listeningplatform.controller
+
+import com.nikitahohulia.listeningplatform.entity.User
+import com.nikitahohulia.listeningplatform.service.UserServiceImpl
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/V1/users")
+class UserController(private val userService: UserServiceImpl) {
+    @GetMapping
+    fun findAllUsers(): ResponseEntity<List<User>> {
+        val users = userService.getAllUsers()
+        return ResponseEntity.ok(users)
+    }
+
+    @GetMapping("/{id}")
+    fun findUserById(@PathVariable id: Long): ResponseEntity<User?> {
+        val user = userService.getUserById(id)
+        return ResponseEntity.ok(user)
```

Comment:
`userService.getUserById(id)` must throw when no such user. Then it would nice to create `@ControllerAdvice` that maps the exception to 404 response status